docs/install/kubernetes.md

```diff
@@ -317,4 +318,64 @@ helm install -n mlrun --values ~/tmp/mlrun-ce-values.yaml mlrun-ce mlrun-ce/mlru
 
 ```{admonition} Note
 If your values have fixed mlrun service versions (e.g.: mlrun:1.3.0) then you might want to remove it from the values file to allow newer chart defaults to kick in
-```
\ No newline at end of file
+```
+
+## Storing artifacts in AWS S3 storage
+
+MLRun CE is using a Minio service as shared storage for artifacts, and accesses it using S3 protocol. This means that
+any path that begins with `s3://` will automatically be directed by MLRun to the Minio service. The default artifact
+path is also configured as `s3://mlrun/projects/{{run.project}}/artifacts` which is a path on the `mlrun` bucket in the
+Minio service.
+
+To store artifacts in AWS S3 buckets instead of the local Minio service, these configurations need to be overridden to 
+make `s3://` paths lead to AWS buckets instead.
+
+```{admonition} Note
+These configurations are only required for AWS S3 storage, due to the usage of the same S3 protocol in Minio. For other
+storage options (such as GCS, Azure blobs etc.) only the artifact path needs to be modified, and credentials need to
+be provided.
+```
+
+### Setting up S3 credentials and endpoint
+
+Set up the following project-secrets (refer to [**Data stores**](../store/datastore.html) and [**Project secrets**](../secrets.html#mlrun-managed-secrets)) 
+for any project used:
+
+* `AWS_ACCESS_KEY_ID`, `AWS_SECRET_ACCESS_KEY` &mdash; S3 credentials
+* `S3_ENDPOINT_URL` &mdash; the AWS S3 endpoint to use, depending on the region. For example: 
+``` console
+S3_ENDPOINT_URL = https://s3.us-east-2.amazonaws.com/
+```
+
+### Disabling auto-mount
+
+Before running any MLRun job that writes to S3 bucket, make sure auto-mount is disabled for it. This can be done in one
+of following ways (refer to [**Disabling auto mount**](../runtimes/function-storage.html#disabling-auto-mount)):
```

Comment:
perhaps give a short brieft what automount is ? just so readers will get the context what is it and _why_ it needs to be disabled?

Reply:
As this is the installation page, I'm only providing a recipe on how to make it work, and provide a link to further details for the interested reader. I did make some changes in the text to make it clearer and changed the link to refer to the entire page about function storage and auto-mount.